primo/opt_model/result_parser.py

```diff
@@ -287,6 +286,8 @@ def get_well_info_dataframe(self):
         """
         Returns the data frame to display in the notebook
         """
+        if self.well_data.config.ignore_operator_name:
+            self._essential_cols.insert(1, self._col_names.operator_name)
         return self.well_data[self._essential_cols]
```

Comment:
little confused at the naming here. If "ignore_operator_name" is true, then we use it? I think there could be a better name for "ignore_operator_name".

Reply:
agree with @strahlw . Would request a change to this option name before the PR can be merged